### PR TITLE
bug/fix-search: Changing search type

### PR DESCRIPTION
**Pull request by andrewrlee**
### Context

https://trello.com/c/AHyLgirl

We were previously doing a term search on a text value.
This has [known limitations](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-term-query.html#avoid-term-query-text-fields)

 This would only work for single word prisons. The text value would be processed during index to split by individual word and lower case whereas the terms query would pass the exact phrase.

So when we searched for 'Cookham wood' it would fail to match either the tokens 'cookham' or 'wood'.
Luckily according to the index I could see that a separate  prison_name.keyword was also indexed which indexes the whole thing.

This change moves to using the formatted name and the keyword field as these currently always match.
We should see if drupal would allow us to index establishment id as going forward a keyword index on establishment id would be more robust.

### Considerations

Worth giving search a bit of a tire kicking generally across each prison

### CODE REVIEW

File: server/routes/__tests__/health.spec.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 const request = require('supertest');
-const express = require('express')
+const express = require('express');
```
Comment (andrewrlee) [inline server/routes/__tests__/health.spec.js:2]: fixing file getting past linter

File: server/routes/health.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 
-const createHealthRouter = ({ buildInfo:build }) => {
+const createHealthRouter = ({ buildInfo: build }) => {
```
Comment (andrewrlee) [inline server/routes/health.js:3]: fixing file getting past linter

**Review by leonk — APPROVED** [review]
LGTM

**Review by davidthomas22 — APPROVED** [review]
Nice work, tested several searches on dev including misspelt words (Exercise, Music and, Guide, guide, bob and, christain, govenor) all returning expected values

**Pull request merged**